querydsl-core/src/main/java/com/querydsl/core/types/dsl/Expressions.java

```diff
@@ -1463,4 +1463,244 @@ private static Template createTemplate(String template) {
         return TemplateFactory.DEFAULT.create(template);
     }
 
+    /**
+     * Create a new BooleanExpression
+     *
+     * @param expr Expression of type Boolean
+     * @return new BooleanExpression
+     */
+    public static BooleanExpression asBoolean(Expression<Boolean> expr) {
+        Expression<Boolean> underlyingMixin = ExpressionUtils.extract(expr);
+        return new BooleanExpression(underlyingMixin) {
+
+            private static final long serialVersionUID = -8712299418891960222L;
+
+            @Override
+            public <R, C> R accept(Visitor<R, C> v, C context) {
+                return this.mixin.accept(v, context);
+            }
+
+        };
+    }
+
+    /**
+     * Create a new BooleanExpression
+     *
+     * @param value boolean
+     * @return new BooleanExpression
+     */
+    public static BooleanExpression asBoolean(boolean value) {
+        return asBoolean(Expressions.constant(value));
```

Comment:
this is inside Expressions, so maybe just `asBoolean(constant(value))`, the same with the others